proto/umee/leverage/v1/tx.proto

```diff
@@ -159,3 +165,20 @@ message MsgSupplyCollateralResponse {
   // Collateralized is the amount of uTokens collateralized.
   cosmos.base.v1beta1.Coin collateralized = 1 [(gogoproto.nullable) = false];
 }
+
+// MsgUpdateRegistry defines the Msg/UpdateRegistry request type.
+message MsgUpdateRegistry {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address of the governance account.
+  string         authority   = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string         title       = 2;
+  string         description = 3;
+  repeated Token registry    = 4 [(gogoproto.nullable) = false];
```

Comment:
Instead of having a new registry list, let's change it to two lists:
* add
* remove

Reply:
Remove is not, and should not be, possibl for the leverage registry. (See EnableSupply, EnableBorrow, and Blacklist behaviors instead).

Add (or update) is straightforward as it is. Please do not change it.

Reply:
Current behavior (in `master` and as is now in this PR) doesn't protect against removing token from the registry. It overwrites everything.
The idea to separate token operations into sub groups makes it more clear and managable. 

That being said, I'm OK, to not support token `removal`. But `add` and `update` should be there, and we should not support registry overwrite.